File: 240204/Next Level/next-level.java
import java.io.*;

class Info {
    String id;
    String level;
    public Info() {
        this.id = "codetree";
        this.level = "10";
    }

    public Info(String id, String level) {
        this.id = id;
        this.level = level;
    }

    public void printUserInfo() {
        System.out.println(String.format("user %s lv %s", this.id, this.level));
    }
}

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input1 = br.readLine().split(" ");
        Info info1 = new Info(input1[0], input1[1]);
        Info info2 = new Info();
        info2.printUserInfo();
        info1.printUserInfo();
    }
}

File: 240317/Next Level/next-level.py
class User:
    def __init__(self, id='codetree', level=10):
        self.id = id
        self.level = level

id, level = input().split()

user1 = User()
user2 = User(id, level)

print(f'user {user1.id} lv {user1.level}')
print(f'user {user2.id} lv {user2.level}')